worker/worker.go

```diff
@@ -223,14 +223,14 @@ type contractCapability struct {
 	RenterKey consensus.PrivateKey
 }
 
-func parseContractCapabilities(contracts []bus.Contract, masterKey [32]byte) []contractCapability {
+func (w *worker) parseContractCapabilities(contracts []bus.Contract) []contractCapability {
```

Comment:
Not really parsing anything here; maybe call it `toCapabilities `.

Alternatively, we could inline this into `withHosts`, since (AFAIK) that's the only place where we need this conversion